### ISSUE TITLE
Patches should have different compounds

### ISSUE THREAD

**Issue by hhyyrylainen** [Revolutionary-Games/Thrive#764]
Patches should have different amounts of compounds and iron in them.

depends on: #683

**Comment by Untrustedlife** [comment]
We will need to set it up so iron spawns in special ways on a per biome basis for this. EG, some biomes should have more iron, some should have less, right now it doesnt make use of this stuff the same way environmental compounds do so i may have to wrap it into biome compounds somehow.

**Comment by Untrustedlife** [comment]
compounds are per biome so that bit will be easy.

**Comment by Untrustedlife** [comment]
It would be nice to have a systemetized version of iron, so we can add more types of chunks in the future without hard coding it.

**Comment by Untrustedlife** [comment]
i need to figure out a way to make iron concentrations vary across biomes.

**Comment by crodnu** [comment]
Adding a variable in the json file and plugging it in the spawner?

**Comment by Untrustedlife** [comment]
yeah, but i need to figure out how to edit /clear previously created spawners.

**Comment by crodnu** [comment]
you can delete one by passing the spawn system the spawner id

**Comment by Untrustedlife** [comment]
Okay i guess its "getting iron to vary by biome" then "patch system" for me

**Comment by Untrustedlife** [comment]
https://github.com/Revolutionary-Games/Thrive/issues/773

**Comment by hhyyrylainen** [comment]
https://github.com/Revolutionary-Games/Thrive/commit/eaf97fde72720bb1cd76752deb438196d3730a58
